api/users/serializers.py

```diff
@@ -40,8 +73,11 @@ def absolute_url(self, obj):
         return obj.absolute_url
 
     def update(self, instance, validated_data):
-        # TODO
-        pass
+        assert isinstance(instance, User), 'instance must be a User'
+        for attr, value in validated_data.items():
```

Comment:
Right, but will this save the social fields correctly in the `social` field of the `User` model? I would think that `validated_data` would have the flattened representation of the social fields, but you need the nested dictionary when saving to the database.